From c7c6070fc7075adf739382bd7f91c5fa833588a5 Mon Sep 17 00:00:00 2001
From: SG <13872653+mmguero@users.noreply.github.com>
Date: Wed, 25 Sep 2019 09:12:39 -0600
Subject: [PATCH] added %DBFIELD% replacer for right-click

Adds %DBFIELD% -> info.dbField replacer for right-click actions (see https://molo.ch/settings#right-click)
---
 viewer/vueapp/src/components/sessions/SessionField.vue | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/viewer/vueapp/src/components/sessions/SessionField.vue b/viewer/vueapp/src/components/sessions/SessionField.vue
index b114ed15..bddcff32 100644
--- a/viewer/vueapp/src/components/sessions/SessionField.vue
+++ b/viewer/vueapp/src/components/sessions/SessionField.vue
@@ -464,6 +464,7 @@ export default {
             .replace('%ISOSTART%', isostart.toISOString())
             .replace('%ISOSTOP%', isostop.toISOString())
             .replace('%FIELD%', info.field)
+            .replace('%DBFIELD%', info.info.dbField)
             .replace('%TEXT%', text)
             .replace('%UCTEXT%', text.toUpperCase())
             .replace('%HOST%', host)
@@ -475,6 +477,7 @@ export default {

           name = (name)
             .replace('%FIELD%', info.field)
+            .replace('%DBFIELD%', info.info.dbField)
             .replace('%TEXT%', text)
             .replace('%HOST%', host)
             .replace('%URL%', url);
@@ -484,6 +488,7 @@ export default {

           value = (value)
             .replace('%FIELD%', info.field)
+            .replace('%DBFIELD%', info.info.dbField)
             .replace('%TEXT%', text)
             .replace('%HOST%', host)
             .replace('%URL%', url);
--
2.20.1

